### PR TITLE
Fix query params and small refactor

### PR DESCRIPTION
**Pull request by jsncmgs1**
Adds a helper function to remove some of the error checking boilerplate,
and loops through files we need to build instead of doing copy pasta.

### CODE REVIEW

**Review by jabrown85 — APPROVED** [review]

File: server/main.go [hunk 1]
```diff
@@ -131,3 +123,9 @@ func HealthHandler(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprint(w, "health check ok")
 	log.Info("health check ok")
 }
+
+func handleError(e error) {
+	if e != nil {
+		log.Error(e)
```
Comment (joshwlewis) [inline server/main.go:129]: log.Error() only sends to logs? Shouldn't these go to rollbar? Also, most of the places we're calling this, an error likely means the following operations won't work. Should we return and abandon further processing when we get these errors?

Comment (jsncmgs1) [reply server/main.go:129]: My understanding is that rollrus adds a hook to the logger that sends to rollbar as well, but I could be wrong: https://github.com/heroku/rollrus/blob/master/examples_test.go

Comment (jsncmgs1) [reply server/main.go:129]: @joshwlewis ^

Comment (jsncmgs1) [reply server/main.go:129]: But to your point - I think log.Panic would make more sense here over log.Error.

**Pull request merged**